tasklauncher/spring-cloud-starter-stream-sink-tasklauncher-cloudfoundry/README.adoc

```diff
@@ -0,0 +1,42 @@
+//tag::ref-doc[]
+= TaskLauncher Cloud Foundry Sink
+
+The TaskLauncher-CloudFoundry sink consumes a `TaskLaunchRequest` message, deploys
+and launches a task using the Cloud Foundry Spring Cloud Deployer.
+
+Any message received by the sink, that does not contain a `TaskLaunchRequest`
+will log an error.
+
+== Using the TaskLauncher
+The task launcher will use the `--uri` from the TaskLaunchRequest to obtain the
+task artifact to deploy, as well as the properties and arguments to execute the
+task.
+An example of this using Spring Cloud Data Flow would be to create a
```

Comment:
Did you intend to add an extra line here (so that all these are different paragraphs)? Seems sensible